pkg/subctl/cmd/show_networks.go

```diff
@@ -52,14 +52,26 @@ func showNetwork(cmd *cobra.Command, args []string) {
 }
 
 func showNetworkSingleCluster(config *rest.Config) {
-	dynClient, clientSet, err := getClients(config)
-	exitOnError("Error creating clients for cluster", err)
+	submariner := getSubmarinerResource(config)
+	if submariner != nil {
+		fmt.Println("    Discovering Network details via Submariner:")
+		clusterNetwork := network.ClusterNetwork{
+			PodCIDRs:      []string{submariner.Status.ClusterCIDR},
+			ServiceCIDRs:  []string{submariner.Status.ServiceCIDR},
+			NetworkPlugin: submariner.Status.NetworkPlugin,
+			GlobalCIDR:    submariner.Status.GlobalCIDR,
+		}
+		clusterNetwork.Show()
+	} else {
+		dynClient, clientSet, err := getClients(config)
+		exitOnError("Error creating clients for cluster", err)
 
-	submarinerClient, err := submarinerclientset.NewForConfig(config)
-	exitOnError("Unable to get the Submariner client", err)
+		submarinerClient, err := submarinerclientset.NewForConfig(config)
+		exitOnError("Unable to get the Submariner client", err)
 
-	clusterNetwork, err := network.Discover(dynClient, clientSet, submarinerClient, OperatorNamespace)
-	exitOnError("There was an error discovering network details for this cluster", err)
+		clusterNetwork, err := network.Discover(dynClient, clientSet, submarinerClient, OperatorNamespace)
+		exitOnError("There was an error discovering network details for this cluster", err)
 
-	clusterNetwork.Show()
+		clusterNetwork.Show()
```

Comment:
I would remove from the other place, and add it here only:

print ( "Discovered network details:")


subctl is already quite verbose, we don't want unnecesary information being printed or it just becomes noise.

Reply:
@mangelajo This is the whole ouput

```
$ ./../submariner-operator/bin/subctl show networks

Showing network details for cluster "cluster1":
    Discovered network details:
        Network plugin:  generic
        Service CIDRs:   [100.1.0.0/16]
        Cluster CIDRs:   [10.1.0.0/16]

Showing network details for cluster "cluster3":
    Discovering Network details via Submariner:
    Discovered network details:
        Network plugin:  weave-net
        Service CIDRs:   [100.3.0.0/16]
        Cluster CIDRs:   [10.3.0.0/16]

Showing network details for cluster "cluster2":
    Discovering Network details via Submariner:
    Discovered network details:
        Network plugin:  weave-net
        Service CIDRs:   [100.2.0.0/16]
        Cluster CIDRs:   [10.2.0.0/16]
```

Reply:
$ ./../submariner-operator/bin/subctl show networks

Showing network details for cluster "cluster1":
    Discovered network details:
        Network plugin:  generic
        Service CIDRs:   [100.1.0.0/16]
        Cluster CIDRs:   [10.1.0.0/16]

Showing network details for cluster "cluster3":
    Discovering Network details via Submariner:
    **Discovered network details:**
        Network plugin:  weave-net
        Service CIDRs:   [100.3.0.0/16]
        Cluster CIDRs:   [10.3.0.0/16]


That's what I mean, see bold ^

Can we try something like: 

```
$ ./../submariner-operator/bin/subctl show networks

Showing network details for cluster "cluster1":
    Discovered network details:
        Network plugin:  generic
        Service CIDRs:   [100.1.0.0/16]
        Cluster CIDRs:   [10.1.0.0/16]

Showing network details for cluster "cluster3":
    Discovering Network details via Submariner:
        Network plugin:  weave-net
        Service CIDRs:   [100.3.0.0/16]
        Cluster CIDRs:   [10.3.0.0/16]

Reply:
Why do we even need to indicate whether or not it came from the Submariner resource? Not sure that adds any value here - that just means it was discovered prior via the same code. I suggest simplifying further:

```
Showing network details for cluster "cluster1":
    Network plugin:  generic
    Service CIDRs:   [100.1.0.0/16]
    Cluster CIDRs:   [10.1.0.0/16]

Showing network details for cluster "cluster3":
    Network plugin:  weave-net
    Service CIDRs:   [100.3.0.0/16]
    Cluster CIDRs:   [10.3.0.0/16]
```

Reply:
I just want to be explicit. It might be easier to debug. But also at the same time avoid code-duplication.